docs/concepts/registries.md

```diff
@@ -199,22 +199,22 @@ Creating Custom Forge Registries
 
 Custom registries can usually just be a simple map of key to value. This is a common style; however, it forces a hard dependency on the registry being present. It also requires that any data that needs to be synced between sides must be done manually. Custom Forge Registries provide a simple alternative for creating soft dependents along with better management and automatic syncing between sides (unless told otherwise). Since the objects also use a Forge registry, registration becomes standardized in the same way.
 
-Custom Forge Registries are created via `RegistryBuilder` via either `RegistryEvent$NewRegistry` or the `DeferredRegister`. The class `RegistryBuilder` takes certain parameters (such as the name, the `Class` of its values, and various callbacks for different events happening on the registry). Calling `RegistryBuilder#create` will result in the registry being built, registered to the `RegistryManager`, and returned to the caller for additional processing.
+Custom Forge Registries are created via `RegistryBuilder` via either `NewRegistryEvent` or the `DeferredRegister`. The class `RegistryBuilder` takes certain parameters (such as the name, the `Class` of its values, and various callbacks for different events happening on the registry). Calling `RegistryBuilder#create` will result in the registry being built, registered to the `RegistryManager`, and returned to the caller for additional processing.
```

Comment:
`RegistryBuilder#create` is package-private now. Both DeferredRegister and NewRegistryEvent call it internally but it cannot be called externally.